data/transport missions.txt

```diff
@@ -1951,3 +1951,383 @@ ship "Science Drone"
+					goto end
+				`	"Sorry, Alondo, but I agree with her cause."`
+			
+			apply
+				set "navy contacted"
+			`	"I understand. Please, just don't drag the Free Worlds into this."`
+			
+			label end
+			`	You say goodbye to Alondo and return to your ship. When you arrive and look around, Diana is notably absent. In your pilot seat she has left her backpack and a written note with the stains of tears in bottom right corner.`
+			``
+			`Dear <first> <last>,`
+			`	I don't care about the credits anymore, but I can't go back to my family. I'll be too much of a disgrace, and I can't even imagine looking my parents in the eyes after what I've done. I've left the credits with you. You can do with them as you wish.`
+			`	I've taken this chance to escape from my old life. I want to become my own person in the world, and I want to do it on my own. Please don't come looking for me, because wherever I am I'll be happier than when I was with my parents. I've failed myself today, but I'm still going to strive for the betterment of the less fortunate. Thank you for trying to make my dreams come true, even if you couldn't succeed.`
+			`	Sincerely,`
+			`				Diana W. Howl`
+			``
+			`	You stare down at the backpack, stuffed full with over two billion credits.`
+			choice
+				`	(Hand the credits over to the Navy.)`
+				`	(Steal the credits.)`
+					goto steal
+			
+			branch navy
+				has "navy contacted"
+			`	You fold the note and slip it into the backpack so that you can show it to the Navy to explain why Diana is not with you. You contact the Navy's crime report hotline and tell them that you have the stolen credits. A few minutes later, they respond telling you that a fleet is waiting on <destination> to retrieve the credits.`
+				accept
+			
+			label navy
+			apply
+				clear "navy contacted"
+			`	You fold the note and slip it into the backpack so that you can show it to the Navy to explain why Diana is not with you. Alondo sends you a message, telling you that the Navy says they are waiting on <destination> for the credits.`
+				accept
+			
+			label steal
+			`	You rip the note apart and begin counting the credits. In total, the backpack contains <payment>. Over two billion credits richer, you sit down and wonder what you will spend it on.`
+				decline
+	
+	on decline
+		payment 2152894400
+		"reputation: Republic" = -1000
+	
+	on complete
+		payment 1000000
+		conversation
+			`	You spot a fleet of docked Navy ships as you come in for a landing. A small squad of troops gather around your ship after you land. You are greeted by a Navy officer upon exitting your ship and explain to him where Diana is, showing him the note she wrote.`
+			`	"That is unfortunate," she responds with a troubled look on her face. "We'll contact the Howl family about this." Before letting you go, the officer hands you <payment>. "This is the reward that Mr. Howl put out for the return of his credits. Thank you for doing the right thing, Captain."`
```

Comment:
maybe use `investments` or `funds` since the payment substitution will add the word credits in.